extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.css

```diff
@@ -40,4 +43,129 @@ amp-story-subscriptions.i-amphtml-story-subscriptions-visible::before {
+}
+
+.i-amphtml-story-subscriptions-signin {
+  display: flex !important;
+  justify-content: center !important;
+  height: 20px !important;
+  margin-bottom: 24px !important;
+  font-size: 12px !important;
+  font-weight: 600 !important;
+}
+
+.i-amphtml-story-subscriptions-signin a {
+  text-decoration: underline !important;
```

Comment:
There is not an underline in the mocks. 
<img width="259" alt="Screen Shot 2022-04-06 at 1 37 48 PM" src="https://user-images.githubusercontent.com/3860311/162045213-d81c7a4c-53d7-48e1-910b-1aacc95d505b.png">